config/default/kustomization.yaml

```diff
@@ -1,16 +1,16 @@
 # Adds namespace to all resources.
-namespace: kamaji-system
+namespace: kamaji-core-system
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: kamaji-
+namePrefix: capi-kamaji-core-
```

Comment:
Although Kamaji plays well with CAPI we have users who are using that outside of this scope, thus, it must be totally unopinionated.